spiffworkflow-frontend/src/routes/OnboardingView.tsx

```diff
@@ -18,11 +19,23 @@ export default function OnboardingView() {
   }, [setOnboarding]);
 
   const onboardingElement = () => {
-    if (onboarding) {
+    if (onboarding && onboarding.instructions.length > 0) {
+      return (
+        <MDEditor.Markdown
+          className="onboarding"
+          linkTarget="_blank"
+          source={onboarding.instructions}
+        />
+      );
+      /*
```

Comment:
Leaving this in for now - as I suspect we want to navigate to the workflow if it is on a user task and is not complete - but need to think more about how to handle this navigation ...